### DISCUSSION TITLE
Is there any way to add custom HTML after the contents?

### DISCUSSION THREAD

**Discussion by leoliu0605** [nunocoracao/blowfish#874] [Q&A] [answered]
As the question in the title, I want to add some custom HTML after every content. I know that Blowfish supports extending the footer and header. Is there a way to extend the content in a similar method?

**Comment by nunocoracao** [comment, marked as answer]
You can use html inside the markdown file

**Reply by leoliu0605** [reply]
Sorry, I mean, is there a way to add a partial HTML section that impacts below or after the content block? In this case, I could adjust one file and it would affect every piece of content.

**Reply by nunocoracao** [reply]
You can override the single layout and add the code you want to change all pages (not lists) - check Hugo's docs for overrides

**Reply by leoliu0605** [reply]
okay, thanks for help
